src/main/views/pending-subscriptions.njk

```diff
@@ -86,9 +86,10 @@
         <p class="govuk-body govuk-!-margin-bottom-8">No pending court subscriptions</p>
       {% endif %}
 
+
       <div class="govuk-!-text-align-left">
+        <a class="govuk-link" href="{{ addOther_link }}">{{ addOther_text }}</a>
```

Comment:
I think the text should be above the button (rather than to the left of it) for 1220. Would you be able to double check?

Reply:
Done :)

Reply:
Hmm still seems to be the same on mine.

<img width="1155" alt="image" src="https://user-images.githubusercontent.com/87066931/168559429-9b99574b-29e6-42c7-a829-33c129d17420.png">

Would you be able to have another look?

Reply:
![Screenshot 2022-05-16 at 10 47 48](https://user-images.githubusercontent.com/23316525/168566250-05880d08-054a-49a9-91e4-57a8d5c88ab9.png)

Wrapped it in a P tag to give it the spacing it needs and added govuk-body so it matches the 19px size of the "Remove" link above.